app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,8 @@
+/*
+ *= require selectize
+ *= require selectize.bootstrap2
+ */
+
```

Comment:
The gem does not give scss imports. Does it work all the same if I use `@import 'selectize'`?